_content/articles/notes-brahmanic-gods-theravadin-cambodia_pou-saveros.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Notes on Brahmanic Gods in Theravadin Cambodia"
+authors:
+  - "Saveros Pou"
+external_url: "http://www.asiainstitutetorino.it/Indologica/volumes/vol14/vol14_art23_POU.pdf"
```

Comment:
This is a good `source_url` but let's use the minified and OCR'ed Archive URL for the main link:  https://archive.org/download/indologica-turinensia-buddhismo/Notes%20on%20Brahmanic%20Gods%20in%20Theravadin%20Cambodia_text.pdf